src/styles/abstracts/_variables.scss

```
@@ -0,0 +1,3 @@
+:root {
+    --button-shadow: 0 0.3px 0.9px rgba(0, 0, 0, 0.32), 0 1.6px 3.6px rgba(0, 0, 0, 0.28);
+}
```

```suggestion
    $button-shadow: 0 0.3px 0.9px rgba(0, 0, 0, 0.32), 0 1.6px 3.6px rgba(0, 0, 0, 0.28);
```
we can use SASS way of doing things